iOSSampleAppWithSDK/SampleAPP/Pods/Manifest.lock

```diff
@@ -0,0 +1,104 @@
+PODS:
```

Comment:
This file seems exact replica of Podfile.lock . Is this really needed ?

Reply:
this generated by cocoapods this no loger there as have added /pods in .gitignore